contracts/bindings/EigenPod/binding.go

```diff
@@ -1,2305 +0,0 @@
-// Code generated - DO NOT EDIT.
```

Comment:
Why is this file getting deleted?

Reply:
Not sure… generating-bindings fails on this file for some weird reason which I haven’t debugged yet.

Reply:
Fixed in [c5284ae](https://github.com/Layr-Labs/eigensdk-go/pull/73/commits/c5284aee3638d34d4925a97a1e39ff27236ba73f)
Super weird bug though... EigenPod needs to be abigen'd right before EigenPodManager otherwise it fails for some unknown reason.. I left comment, not going to spend time debugging this.